swagger/v1/openapi.json

```diff
@@ -1078,6 +1078,160 @@
         }
       }
     },
+    "/value_definitions": {
+      "get": {
+        "summary": "List all value definitions",
+        "tags": [
+          "value_definition"
+        ],
+        "description": "Lists all value definitions requested",
+        "operationId": "ListValueDefinitions",
+        "parameters": [
+          {
+            "name": "X-RH-IDENTITY",
+            "in": "header",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "limit",
+            "in": "query",
+            "required": false,
+            "description": "The number of items to return",
+            "schema": {
+              "type": "integer",
+              "maximum": 100,
+              "minimum": 1,
+              "default": 10
+            }
+          },
+          {
+            "name": "offset",
+            "in": "query",
+            "required": false,
+            "description": "The number of items to skip before starting to collect the result set",
+            "schema": {
+              "type": "integer",
+              "minimum": 1,
+              "default": 1
+            }
+          },
+          {
+            "name": "search",
+            "in": "query",
+            "required": false,
+            "description": "Query string compliant with scoped_search query language: https://github.com/wvanbergen/scoped_search/wiki/Query-language",
```

Comment:
@skateman you could use Markdown/HTML to make this link clickable, take a look at https://github.com/RedHatInsights/compliance-backend/blob/f4aec130851026bff95712c09f3c9336a470886e/swagger/v1/openapi.json#L3486

Reply:
This is a global issue in the whole JSON, so please deal with it separately.